### PR TITLE
Persons for ef9524

### PR DESCRIPTION
**Pull request by DariaElagina**
Elaborated exsting entries: corrected the transliteration, added information, relations and bibliography.

### CODE REVIEW

File: PRS6001-7000/PRS6567Makwanne.xml [hunk 1]
```diff
@@ -51,19 +51,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                 wife gave birth to their son <persName ref="PRS10309YelmaM"></persName>. In <date>1876</date>, he married
+                 <persName ref="PRS10255Yasimma">Yašimmabet ʾAli</persName>, who gave birth to <persName ref="PRS5226haylase">Tafari</persName>.
+                 In <date>1876</date> Makʷannǝn Walda Mikāʾel became <roleName type="rank">bālāmbārās</roleName> and in
+                 <date>1881</date> he joined a campaign against the <persName ref="ETH1973Oromo"></persName>.
+                 After the conquest of <placeName ref="LOC3786Harar"></placeName>, he was awarded the rank of
+                 <roleName type="rank">daǧǧāzmāč</roleName> and became the governor of the region. In <date>1887</date>
+                 Makʷannǝn Walda Mikāʾel started the imperial expansion to the east and south of
+                 <placeName ref="LOC3786Harar"></placeName> and conquered the parts of <placeName ref="LOC5040Ogaden"></placeName>,
+                 as well as parts of the territories of <persName ref="ETH1496Gadabu"></persName> and
+                 <persName ref="ETH1728Issa"></persName>.<lb/>
```
Comment (thea-m) [inline PRS6001-7000/PRS6567Makwanne.xml]: In the guidelines, `<lb>` is only mentioned in text encoding. Maybe a structuring in `<p>`s (to be visualized, if they are not yet...) would be more meaningful?

File: PRS10001-11000/PRS10255Yasimma.xml [hunk 2]
```diff
@@ -51,23 +51,36 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-               <persName>
-                  
-                  Yäšimmäbet Ali
-               </persName>
+              <persName xml:id="n1" xml:lang="gez">የሺመቤት፡ አሊ፡</persName>
+               <persName corresp="#n1" xml:lang="gez" type="normalized">Yašimmabet ʾAli</persName>
+               <birth notBefore="1864" notAfter="1866">She was a daughter of <persName ref="PRS9833Walatta"></persName> and
+               <persName ref="PRS1683AliAbba"></persName>.
+             </birth>
+               <floruit>In <date>1894</date> at the age of 11 she married <persName ref="PRS6567Makwanne"></persName>
```
Comment (thea-m) [inline PRS10001-11000/PRS10255Yasimma.xml]: To me it seems a comma after 1894 or 11 (should be eleven?) would be nice.

Comment (DariaElagina) [reply PRS10001-11000/PRS10255Yasimma.xml]: I added a comma. As far as I could check small numbers ranging from one to ten should generally be spelled out. Larger numbers (i.e., above ten) are written as numerals, unless they are at the verz beginning of a sentence.

**Comment by thea-m** [comment]
In case I restored, I didn't want to, I just accidentally clicked twice on delete...

**Pull request merged**